src/main/controllers/LiveStatusController.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from 'express';
+import moment from 'moment';
+import { LiveCasesActions } from '../resources/actions/liveCaseActions';
+
+export default class LiveStatusController {
+  public get(req: Request, res: Response): void {
+    const courtId = req.query.courtId as string;
+    const timerMilliseconds = process.env.REFRESH_TIME_MILLISECONDS ? process.env.REFRESH_TIME_MILLISECONDS : 15000;
```

Comment:
default values can live in the declaration of the environment variable, that way we only need to change default in 1 place

Reply:
This is the only way I can set env variable on Windows machine

Reply:
we cant see what your variables are though? sensitive info isnt stored in the env file, wed have secrets placeholders so it would look something like:
```
REFRESH_TIME_MILLISECONDS=${REFRESH_TIME} | 15000
```